app/components/TransactionHistory/TransactionHistoryPanel/TransactionHistoryPanel.scss

```diff
@@ -22,3 +22,7 @@
 .oddNumberedRow {
   background: #f8f8f8;
 }
+
+.transactions {
+  margin: -24px;
```

Comment:
Moved from the now generic `Transactions.scss .transactionList` to the more specific `TransactionHistoryPanel.scss .transactions` which it's meant for.